scripts/run_validator_node.sh

```diff
@@ -0,0 +1,238 @@
+#!/bin/bash
+
+set -ue
+set -o pipefail
+
+sudo_cmd=""
+files_path=""
+chain_spec_name=""
+keychain_exists=""
+session_keys=""
+binary_path=""
+
+echo "This script will setup a Atleta Validator on your PC. Press Ctrl-C at any time to cancel."
+echo "Checking privileges... "
+if [ $(id -u) -eq 0 ]; then
+    echo "OK (root)"
+elif command -v sudo &> /dev/null; then
+    echo "OK (sudo)"
+    sudo_cmd=sudo
+else
+    echo "FAIL"
+    echo "Not running as root and no sudo detected. Please run this as root or configure sudo. Exiting." >&2
+    exit 1
+fi
+
+echo "Detecting your architecture... "
+operating_system=$(uname -s)
+if [ "$operating_system" != "Linux" -a "$operating_system" != "Darwin" ]; then
+    echo "$operating_system is currently not supported by Atleta Node, only Linux or Darwin are supported. Exiting." >&2
+    exit 1
+else
+    echo "OK ($operating_system)"
+fi
+
+if [[ $operating_system == "Linux" ]]; then
+    echo -n "Checking for systemd... "
+    if [ -e /run/systemd/system ]; then
+        echo "OK"
+    else
+        echo "FAIL"
+        echo "No systemd detected. Exiting." >&2
+        exit 1
+    fi
+fi
+
+if ! command -v jq &>/dev/null || \
```

Comment:
Split this into two check. Then it will be possible to point what exact requirement is missing. Also in the future another requirement can comes into scope.